mount/mount_windows.go

```diff
@@ -105,14 +105,11 @@ func (mounter *Mounter) MountSensitive(source string, target string, fstype stri
 			klog.Warningf("SMB Mapping(%s) returned with error(%v), output(%s)", source, err, string(output))
 			if isSMBMappingExist(source) {
 				valid, errPath := isValidPath(source)
-				if errPath != nil {
-					return errPath
-				}
-				if valid {
+				if valid && errPath == nil {
 					err = nil
 					klog.V(2).Infof("SMB Mapping(%s) already exists and is still valid, skip error", source)
 				} else {
-					klog.V(2).Infof("SMB Mapping(%s) already exists while it's not valid, now begin to remove and remount", source)
+					klog.V(2).Infof("SMB Mapping(%s) already exists while it's not valid, return error: %v, now begin to remove and remount", source, errPath)
```

Comment:
Could `errPath` be some temporary error that resolves itself on the next call?

Does removing the global mapping impact running containers?

Reply:
it could be temp error, so pod accessing that path at that time would still get error, so remove the global mapping has the same impact since at that time, that path is inaccessible either

Reply:
Could it be some temporary error that only impacts this particular call, like a timeout or network issue?

Reply:
if it's a temporary error, remove and create new connection is also ok, since at that time, connection was broken.

Reply:
Ok I'm mainly concerned that if there's some temporary error and we cannot reconnect for some reason, then we'll be breaking all existing mounts.

Typically a mount for one Pod should not impact mounts for running Pods.

Reply:
In the case of password change scenario, there are two options
1 remove old connection, make new connection so allowing new pods work, but it seems affect existing ones. Or
2. allow new pods fail while keeping existing ones work.

Similar, for other type of errors, we might take different options based on the error.

Reply:
if password changed, existing connection would be invalid, remove old connection would be the only option.

Reply:
Windows smb mount is different than Linux, on Windows, the smb mount is shared, so at that time, if connection is broken, all running pods using that smb mount are broken too, I think remove and create new connection is ok.

Reply:
Thanks, Andy. I think the only concern left is how reliable of validating path is. For example, in case of some temporal network error, the checking failed. In the case, we want to keep the existing mount connection. Is that possible to identify such temporal network error?

Reply:
thanks, I did a change first, remove old connection and create new connection only happens when there is `Access is denied` error

Reply:
is there any other type of error we are sure that old connection will not work, and better to delete it and create a new one?

Reply:
> is there any other type of error we are sure that old connection will not work, and better to delete it and create a new one?

@jingxu97 I don't know, let's fix this error type first.